modules/core/src/main/java/org/apache/ignite/internal/sql/command/SqlCreateUserCommand.java

```diff
@@ -36,6 +36,22 @@
     /** User's password. */
     private String passwd;
```

Comment:
Let's annotate field  with @IgniteToStringExclude to avoid being logged unintentionally.